.gitignore

```diff
@@ -2,4 +2,5 @@
 \#*
 *.prof
 tmp-build
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea
```

Comment:
How is this related to go modules?

Reply:
Not very related just to ignore internal goland IDE and mac folders to eliminate mistakes, I can remove them if its not according to the standards, do you want me to revert this one?